addons/project/wizard/project_share_wizard.py

```diff
@@ -66,19 +66,54 @@ def create(self, vals_list):
         wizards = super().create(vals_list)
         for wizard in wizards:
             collaborator_ids_to_add = []
-            collaborator_ids_to_remove = []
+            collaborator_ids_to_add_with_limited_access = []
+            project_collaborator_ids_to_remove = []
```

Comment:
I added `project` in the variable name because we will remove `project.collaborator` record and not just a line inside that wizard. 😅 

But yeah I can remove `project_` in the name if you prefer. 😁